ft_prepare_mesh.m

```diff
@@ -210,6 +211,13 @@
   end
 end
 
+% smooth the mesh
+if ~isempty(cfg.smooth)
+  cfg.headshape = bnd;
+  cfg.numvertices = [];
+  bnd = prepare_mesh_headshape(cfg);
```

Comment:
it would be better to move the smoothing out of the prepare_mesh_headshape function. But for now this is fine.